<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaneImportedReportFiles</class>
 <widget class="QWidget" name="PaneImportedReportFiles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Imported report files</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonRemove">
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonExpandAll">
       <property name="toolTip">
        <string>Étendre afin de voir tous les fichiers</string>
       </property>
       <property name="text">
        <string>Étendre</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSort">
       <property name="toolTip">
        <string>Trier tous les fichiers par order alphabétique</string>
       </property>
       <property name="text">
        <string>Trier</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeViewFiles">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
